tools/test_vlanifns_primitive.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+from cloudcix_primitives import vlanifns
+
+# Run the following test scripts before this one:
+#
+# * `tools/test_directorymain_primitive.py build /etc/netns/mynetns` to ensure the directories needed
+#   are in place.
+# * `tools/test_ns_primitive.py build mynetns to ensure the name space we want to run dhcpns in exists
+
+cmd = sys.argv[1]
+
+namespace = 'mynetns'
+
+vlan = 4000
+ifname = 'private0'
+
+
+if len(sys.argv) > 2:
```

Comment:
```suggestion
if len(sys.argv) > 2:
    namespace = sys.argv[2]
if len(sys.argv) > 3:
    vlan = sys.argv[3]
if len(sys.argv) > 4:
    ifname = sys.argv[4]
```

Should it be extended to allow `vlan` and `ifname` to be arguments in the test request? 

e.g `tools/test_ns_primitive.py build mynetns 4000 private0`